FORKED-WORKFLOW.md

```diff
@@ -0,0 +1,31 @@
+# Apollo iOS Forked Workflow Guide
+
+When working with Apollo iOS and forking the repositories, there are some scenarios that may arise in which you have questions on the best way to proceed. This guide will collect these questions and aim to provide clarity for these different scenarios.
+
+This guide assumes you have read the [CONTRIBUTING](https://github.com/apollographql/apollo-ios-dev/blob/main/CONTRIBUTING.md) guide to get an overview of how our repositories are structured and worked on. Please submit an issue to the [apollo-ios](https://github.com/apollographql/apollo-ios/issues) repository.
```

Comment:
```suggestion
This guide assumes you have read the [CONTRIBUTING](https://github.com/apollographql/apollo-ios-dev/blob/main/CONTRIBUTING.md) guide to get an overview of how our repositories are structured and worked on. Please submit issues to the [apollo-ios](https://github.com/apollographql/apollo-ios/issues) repository.
```

Reply:
I assume this is the general "please submit issues over there, not here" message; plural "issues" reads better in that case.